main.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"fmt"
+)
+
+func Even(a uint64) bool {
+	a = a << 63
```

Comment:
Looks good, but `a & 1 == 0` is slightly better, because you don't need to change `63` constant once you change the type.